spanner/transaction.go

```diff
@@ -853,6 +890,33 @@ func (t *ReadOnlyTransaction) release(err error) {
 	}
 }
 
+func errCannotSetDirectedReadOptions() error {
+	return spannerErrorf(codes.InvalidArgument, "DirectedReadOptions cannot be set for ReadWriteTransaction or PDML")
```

Comment:
This is a bad request error not invalid arguement

Reply:
`BadRequest` as a code is not available as part of grpc codes.
Can you please suggest which one would be more suitable from the available one's (https://github.com/grpc/grpc-go/blob/master/codes/codes.go) as this is what golang client uses.

Reply:
As discussed offline, we will be using `InvalidArgument` only